src/test/resources/log4j.properties

```diff
@@ -0,0 +1,50 @@
+#
+# ZooKeeper Logging Configuration
+#
+
+# Format is "<default threshold> (, <appender>)+
+
+# DEFAULT: console appender only
+log4j.rootLogger=INFO, CONSOLE
+#log4j.rootLogger=INFO, ROLLINGFILE
+
+# Example with rolling log file
+#log4j.rootLogger=DEBUG, CONSOLE, ROLLINGFILE
+
+# Example with rolling log file and tracing
+#log4j.rootLogger=TRACE, CONSOLE, ROLLINGFILE, TRACEFILE
+
+#
+# Log INFO level and above messages to the console
+#
+log4j.appender.CONSOLE=org.apache.log4j.ConsoleAppender
```

Comment:
does logging configuration belong to this component? it should be defined for zk, not for authorizer, which is plugin.

Reply:
This is for the unit tests only. We need log setting so we can see the logs in stdout when we run the unit tests